industrial_ci/src/ci_main.sh

```diff
@@ -39,6 +39,9 @@ if [ "$PRERELEASE" == true ]; then
 elif [ -n "$ABICHECK_URL" ]; then
   source ${ICI_SRC_PATH}/tests/abi_check.sh
   run_abi_check
+elif [ -f "$TARGET_REPO_PATH/.clang-format" -o -n "$CLANGCHECK" ] && [ "$CLANGCHECK" != "false" ]; then
```

Comment:
This would prevent all other tests in case `.clang_format` is present.
I see two options:

1. Use `CLANG_FORMAT=<style>|file` to eable the test. In case of "file", specifiy the absolute path to `.clang_format`.
2. Move into `source_tests.sh`

Option 1 would enable to allow failures regarding the format.
Option 2 would allow the auto-check (with opt-out).